notebooks/community/feature_store/mobile_gaming_feature_store.ipynb

```diff
@@ -0,0 +1,2274 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        feature_ids = customer_entity_type.batch_create_features(
</pre><p>The return type hint for <code>batch_create_features</code> is <code>EntityType</code>, it would be misleading as returning <code>feature_ids</code> and call <code>list_features()</code>. The below snippets will work.</p><p> </p><p> </p><p><code>try:</code></p><p><code>  customer_entity_type.batch_create_features(</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;feature_configs = feature_configs,</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;sync=True)</code></p><p><code>except RuntimeError as error:</code></p><p><code>&nbsp;&nbsp;print(error)</code></p><p><code>else:</code></p><p><code>&nbsp;&nbsp;for feature in customer_entity_type.list_features():</code></p><p><code>&nbsp;&nbsp;&nbsp;print('')</code></p><p><code>&nbsp;&nbsp;&nbsp;print(f'The resource name of {feature.name} feature is', feature.resource_name)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>